controller.js

```diff
@@ -0,0 +1,33 @@
+const input =$('#input')
+const adPost= function(){
+    tweeter.addPost(input.val())
+    input.val('')
+
+    Renderer()
+}
+$('#posts').on('click','.commentAdd', function() {
+    let pstId=$(this).closest(".post").attr('id')
```

Comment:
here you can use `const` because you are not edditing the value of `pstId` again in that function.